javascript/adonisjs-slim/tests/functional/hello_world.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { test } from '@japa/runner'
```

Comment:
no need to add specs at framework level (there is one at global level that run on each framework)

Reply:
Understood.  These are automatically added by the framework when creating a new project.